network/load-balancer/how-to/manage-frontends-and-backends.mdx

```diff
@@ -27,12 +27,12 @@ When you [create your Load Balancer](/network/load-balancer/how-to/create-load-b
 1. Click **Load Balancers** in the **Network** section of the [Scaleway Console](https://console.scaleway.com) side menu. The Load Balancer dashboard displays.
 2. Click the Load Balancer you want to add frontends to. The Load Balancer information page displays.
 3. Click the **Frontends** tab.
-4. Click the green **+** button.
+4. Click the <Icon name="plus" /> button.
     <Lightbox src="scaleway-add-frontends.webp" />
 5. Enter a **name**, **port** and (if necessary) [**SSL certificate(s)**](/network/load-balancer/how-to/add-certificate) for your new frontend. Note that each frontend must listen on a different port. For help configuring your frontend, see our [dedicated reference documentation](/network/load-balancer/reference-content/configuring-load-balancer).
 6. Choose between **using an existing backend** or **creating a new backend**.
     - If using an existing backend, click **Use an existing Backend** and select the backend of your choice from the dropdown list.
-    - If creating a new backend, click **Create a new Backend** and enter the required information and advanced settings as required. For help configuring your backend, see our [dedicated reference documentation](/network/load-balancer/reference-content/configuring-load-balancer)
+    - If creating a new backend, click **Create a new Backend** This will open a new tab in your browser for you to configure your backend. Enter the required information and advanced settings as required.. For help configuring your backend, see our [dedicated reference documentation](/network/load-balancer/reference-content/configuring-load-balancer)
```

Comment:
```suggestion
    - If creating a new backend, click **Create a new Backend**. This will open a new tab in your browser for you to configure your backend. Enter the required information and advanced settings as required. For help configuring your backend, see our [dedicated reference documentation](/network/load-balancer/reference-content/configuring-load-balancer)
```